lib/lago/adyen/error_handlable.rb

```diff
@@ -4,13 +4,15 @@ module Lago
   module Adyen
     module ErrorHandlable
       def handle_adyen_response(res)
-        return if res.status < 400
+        return true if res.status < 400
 
         code = res.response['errorType']
         message = res.response['message']
 
-        deliver_error_webhook(::Adyen::AdyenError.new(nil, nil, message, code))
-        result.service_failure!(code:, message:)
+        error = ::Adyen::AdyenError.new(nil, nil, message, code)
+        deliver_error_webhook(error)
+
+        false
```

Comment:
We are loosing the ability to track the initial error message from the service. Is it intended?

Reply:
I added the error tracking.